### ISSUE TITLE
Cannot read property 'info' of undefined

### ISSUE THREAD

**Issue by xbill82** [webdriverio/webdriverio#1157]
Hello everybody,
I'm trying wdio v4 with my test suite, here's the stack

* wdio v4.0.3
* wdio-cucumber-framework v0.0.5
* phantomjs v2.1.1
* my [wdio.conf.js](http://codepen.io/xbill82/pen/mPPyab?editors=0010)

This produces two weird behaviors

* every feature starts a new wdio session (I didn't expect this but maybe it's the way it is supposed to work?)
* all the spawned sessions crash with the following error (quite cryptic, would be awesome to understand what it is caused by)

```
/var/app/node_modules/webdriverio/node_modules/q/q.js:141
                throw e;
                ^

TypeError: Cannot read property 'info' of undefined
    at Object.reject (/var/app/node_modules/webdriverio/build/lib/webdriverio.js:287:20)
    at Object.callErrorHandlerAndReject (/var/app/node_modules/webdriverio/build/lib/webdriverio.js:217:27)
    at /var/app/node_modules/webdriverio/build/lib/webdriverio.js:192:58
    at run (/var/app/node_modules/webdriverio/node_modules/core-js/library/modules/es6.promise.js:104:47)
    at /var/app/node_modules/webdriverio/node_modules/core-js/library/modules/es6.promise.js:115:28
    at flush (/var/app/node_modules/webdriverio/node_modules/core-js/library/modules/$.microtask.js:19:5)
    at doNTCallback0 (node.js:417:9)
    at process._tickCallback (node.js:346:13)
```

**Comment by christian-bromann** [comment]
Fixed by https://github.com/webdriverio/webdriverio/commit/2a36579e6b5bf92734c3a1bb31361dd072ac4078 will be published with the next version v4.0.4

**Comment by xbill82** [comment]
Great job guys, looking forward for the release on npm. Keep it up!
